typescript/infra/scripts/print-gas-prices.ts

```diff
@@ -40,14 +40,27 @@ async function main() {
 
   const prices: ChainMap<GasPriceConfig> = Object.fromEntries(
     await Promise.all(
-      supportedChainNames.map(async (chain) => [
-        chain,
-        await getGasPrice(
-          mpp,
-          chain,
-          gasPrices[chain as keyof typeof gasPrices],
-        ),
-      ]),
+      supportedChainNames.map(async (chain) => {
+        try {
+          return [
+            chain,
+            await getGasPrice(
+              mpp,
+              chain,
+              gasPrices[chain as keyof typeof gasPrices],
+            ),
+          ];
+        } catch (error) {
+          console.error(`Error getting gas price for ${chain}:`, error);
```

Comment:
curious why motivated this? do we need it?

Reply:
yes because as we add more chains we may not have 100% reliability of the getGasPrice function due to the variability of RPCs

therefore by try-catching we log the failures but still log the updated gasprices for chains that didnt fail

Reply:
notably encountered issues with infintyvm, tangle, glue recently